tests/analog/test_annealing_program.py

```diff
@@ -118,7 +118,7 @@ def test_transpile(self, annealing_program, transpiled_program_dictionary):
 
     def test_get_waveforms(self, annealing_program_transpiled):
         """Test get waveforms method works as intended"""
-        anneal_waveforms = annealing_program_transpiled.get_waveforms()
+        anneal_waveforms = annealing_program_transpiled.get_waveforms(minimum_clock_time=1)
```

Comment:
Could you add some more tests with different `minimum_clock_time` values?

Testing only the default value could be risky